src/main/java/com/tstepnik/planner/domain/validators/BeforeNowValidator.java

```diff
@@ -0,0 +1,27 @@
+package com.tstepnik.planner.domain.validators;
+
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+public class BeforeNowValidator implements ConstraintValidator<NotBeforeNow, ZonedDateTime> {
```

Comment:
Useless since there are ready to use java validators. Good job with creating your own though.

Reply:
Yeah, I know it's validator like future or present but as you  thought i wanted to create my own :)